src/audio/pipewire/SDL_pipewire.c

```diff
@@ -1100,7 +1100,7 @@ PIPEWIRE_OpenDevice(_THIS, void *handle, const char *devname, int iscapture)
     }
 
     /* Load the rtkit module so Pipewire can set the loop thread to the appropriate priority */
-    props = PIPEWIRE_pw_properties_new(PW_KEY_CONTEXT_PROFILE_MODULES, "default,rtkit", NULL);
+    props = PIPEWIRE_pw_properties_new(PW_KEY_CONFIG_NAME, "client-rt.conf", NULL);
```

Comment:
Is it possible to check the new property, then the old property if the new one doesn't exist? We should be able to self-define PW_KEY_CONFIG_NAME if 0.3.20 doesn't define it. If this is possible we might be able to keep the version requirement down, other than that I don't see any issues here!

Reply:
Or, if newer versions of Pipewire ignore the old property and older versions ignore the new property, would it make sense to set *both* properties?